css/css-variables-1/variable-definition.html

```diff
@@ -0,0 +1,104 @@
+<!DOCTYPE html>
+<html>
+<head>
+    <title>variable-definition: get variable value using getPropertyValue</title>
+
+    <meta rel="author" title="Kevin Babbitt">
+    <meta rel="author" title="Greg Whitworth">
+    <link rel="author" title="Microsoft Corporation" href="http://microsoft.com" />
+    <link rel="help" href="http://www.w3.org/TR/css-variables-1/#using-variables">
+
+    <script src="/resources/testharness.js"></script>
+    <script src="/resources/testharnessreport.js"></script>
+</head>
+<body>
+    <div id="log"></div>
+
+<script type="text/javascript">
+    "use strict";
+
+    let templates = [
+        { varName:"--var",     expectedValue:"",        style:"",                           testName:"no variable"},
+        { varName:"--var",     expectedValue:"value",   style:"--var:value",                testName:"variable"},
+        { varName:"--v",       expectedValue:"value",   style:"--v:value",                  testName:"single char variable"},
+        { varName:"---",       expectedValue:"value",   style:"---:value",                  testName:"single char '-' variable"},
+        { varName:"--",        expectedValue:"value",   style:"--:value",                   testName:"no char variable"},
+        { varName:"--var",     expectedValue:" ",       style:"--var: ",                    testName:"white space value (single space)"},
+        { varName:"--var",     expectedValue:"  ",      style:"--var:  ",                   testName:"white space value (double space)"},
+        { varName:"--var",     expectedValue:"value2",  style:"--var:value1; --var:value2", testName:"overwrite"},
+        { varName:"--var",     expectedValue:"value",   style:"--var:value;--var:;",        testName:"can't overwrite with no value"},
+        { varName:"--var",     expectedValue:" ",       style:"--var:value;--var: ;",       testName:"can overwrite with space value"},
+        { varName:"--var",     expectedValue:"value1",  style:"--var:value1; --Var:value2", testName:"case sensetivity"},
+        { varName:"--Var",     expectedValue:"value2",  style:"--var:value1; --Var:value2", testName:"case sensetivity"},

```

Comment:
Some test name as the line above, also that's not how you spell sensitivity. See the rest of the unstable results found.